test-common/src/main/java/com/mentoring/database/DbSelects.java

```diff
@@ -51,23 +55,22 @@ public static String getAccessTokenIdByEmailAddress(Database connection, String
         return result;
     }
 
-    public static String getCharacterIdByCharacterName(Database connection, String characterName) throws SQLException {
-        String result = connection.runSelectStringValue(
-                CHARACTER_ID_BY_CHARACTER_NAME,
-                connection.getSingleStringMapper(1, characterName),
-                TableColumn.SKYXP_CHARACTER__CHARACTER_ID
-        );
-        log.info("Query = {};\tparam = {};\tresult = {}", CHARACTER_ID_BY_CHARACTER_NAME, characterName, result);
-        return result;
```

Comment:
You have deleted this while it was being used. If you are going to replace this with getCharacterIdByCharacterNameAndUserId, then you need to refactor it's usages as well, now it's failing with compile error.

Reply:
Aaah, it was late night - as usual :-) Thanks! Refactored because the getCharacterIdByCharacterName is not OK as we discussed earlier.